controllers/AdminController.ts

```diff
@@ -38,12 +39,20 @@ export default class AdminController implements AdminControllerI {
                 AdminController.adminController.findAllUsers); 
             app.put("/admin/api/users/:uid/block",
                 AdminController.adminController.blockUser); 
+            app.put("/admin/api/users/:uid/unblock",
+                AdminController.adminController.unblockUser); 
             app.post("/admin/api/users",
                 AdminController.adminController.createUser);
             app.delete("/admin/api/users/:uid",
                 AdminController.adminController.deleteUser);
-            app.put("/api/users/:userid", 
+            app.get("/admin/api/tuits",
+                AdminController.adminController.findAllTuits);
+            app.put("/admin/api/users/:userid", 
                 AdminController.adminController.updateUser);
+            app.put("/admin/api/tuits/:tid", 
+                AdminController.adminController.updateTuit);
+            app.get("/admin/api/users/:uid",
+                AdminController.adminController.findUserById);
```

Comment:
This should be adminController.findByUsername

The reasoning is that when a user is being created, they don't have an ID for us to use. We need to compare something else and username or email is usually the best choice. 

Our DAO would then be UserModel.findOne({username: username}) 

Otherwise thank you for accommodating the other changes!